tests/tests/cpld_control_sfs_180d

```diff
@@ -0,0 +1,116 @@
+#
+#  cpld_control GFSv17 with UGWPv1 test, i.e., GSL orographic drag +
+#  version 1 non-stationary GWD, hydrostatic, long forecast
+#
+
+export TEST_DESCR="Fully coupled FV3-CCPP-MOM6-CICE-CMEPS system hydrostatic version - C96MX100 - long forecast"
+
+export CNTL_DIR=cpld_control_sfs_180d
+
+export LIST_FILES="sfcf8280.nc \
+                   sfcf8640.nc \
+                   atmf8280.nc \
+                   atmf8640.nc \
+                   GFSFLX.GrbF8280 \
+                   GFSFLX.GrbF8640 \
+                   GFSPRS.GrbF8280 \
+                   GFSPRS.Grb8640 \
+                   RESTART/20220317.060000.MOM.res.nc" 
+
+export_fv3
+export_cpl
+export_ugwpv1
+
+export CPLCHM=.false.
+export DNATS=0
+export MAKE_NH=.false.
+export DDDMP=0
+export DO_VORT_DAMP=.false.
+export HYDROSTATIC=.true.
+export HORD_XX=10
+export HORD_DP=10
+export KORD_XX=12
+export KORD_TM=-12
+export D_CON=0
+
+export RESTART_N=360
+export RESTART_INTERVAL="${RESTART_N} -1"
+export OUTPUT_FH='0 8280 8640'
+export OUTPUT_GRID="'gaussian_grid'"
+export WRITE_DOPOST=.true.
+export MOM6_OUTPUT_FREQ=360
+export MOM6_OUTPUT_UNITS=days
+
+export FHMAX=8640
+
+export INPES=$INPES_cpl_bmrk
+export JNPES=$JNPES_cpl_bmrk
+export WRTTASK_PER_GROUP=$WPG_cpl_bmrk
+
+OCN_tasks=$OCN_tasks_cpl_bmrk
+ICE_tasks=$ICE_tasks_cpl_bmrk
+WAV_tasks=0
```

Comment:
@LarissaReames-NOAA you may want to export these. If I understand, turning off wave is cheaper but then this configuration is more of a nowave_noaero test

It may also be worth adding more files to the baseline in LIST_FILES (CICE, history files,...) if this would be running anyways